src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -75,4 +97,71 @@ class LocalBackend extends Backend {
       taskOutput.name -> rawValue.map { possiblyAutoConvertedValue(taskOutput, _) }
     }.toMap
   }
+
+  /**
+   * Creates host execution directory, inputs path, and outputs path.  Stages any input files into the workflow-inputs
+   * directory and localizes their paths relative to the container.
+   */
+  override def initializeWorkflow(workflow: WorkflowDescriptor, actualInputs: Map[FullyQualifiedName, WdlValue]): Map[FullyQualifiedName, WdlValue] = {
+    val hostExecutionDirectory = hostExecutionPath(workflow).toFile
+    hostExecutionDirectory.mkdirs()
+
+    val hostExecutionAbsolutePath = hostExecutionDirectory.getAbsolutePath
+    val hostInputsPath = Paths.get(hostExecutionAbsolutePath, "workflow-inputs")
```

Comment:
We could do that.  All the other directories are named `call-ps` and the like, I wanted to make it clear these were the workflow inputs.  And BTW I don't currently have a use for workflow outputs...